src/libsync/theme.cpp

```diff
@@ -117,7 +117,7 @@ QString Theme::statusHeaderText(SyncResult::Status status) const
 
 bool Theme::isBranded() const
 {
-    return appNameGUI() != QStringLiteral("Nextcloud");
+    return (appNameGUI() != QStringLiteral("Nextcloud") && appNameGUI() != QStringLiteral("NextcloudDev"));
```

Comment:
if possible, it would be better to use the macro `NEXTCLOUD_DEV` instead of using the string comparison